### DISCUSSION TITLE
Deleted readme.md of profile

### DISCUSSION THREAD

**Discussion by AndreaZavatta** [community/community#40012] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I accidentally deleted my profile readme.me, how can I retrieve it?
I tried following the guide on how to retrieve a deleted repository, but the readme doesn't seem to see it as a repository.
PLEASE HELP

**Comment by ossanna16** [comment]
Hi there @AndreaZavatta and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by futuresea-dev** [reply]
Thank you
